sync/syncer.go

```diff
@@ -224,17 +223,13 @@ func (s *Syncer) run() {
 			return
 		case <-s.forceSync:
 			go syncRoutine()
-		case layer := <-s.LayerCh:
-			s.currentLayerMutex.Lock()
-			s.currentLayer = layer
-			s.currentLayerMutex.Unlock()
-			s.Debug("sync got tick for layer %v", layer)
+		case <-s.syncTimer.C:
```

Comment:
There couldn't be more than one syncRoutine right?

Reply:
nope